unicode_smilies/unicode_smilies.php

```diff
@@ -862,6 +679,7 @@ function unicode_smilies_smilies(&$a,&$b) {
 	Smilies::add($b, ':cookie:', '&#x1F36A');
 	Smilies::add($b, ':birthday cake:', '&#x1F382');
 	Smilies::add($b, ':shortcake:', '&#x1F370');
+    Smilies::add($b, ':cake:', '&#x1F370;');
```

Comment:
Code standards: Please normalize indentation.
```suggestion
	Smilies::add($b, ':cake:', '&#x1F370;');
```